packages/web-react/src/components/Toast/stories/UncontrolledToast.stories.tsx

```diff
@@ -64,6 +67,12 @@ const meta: Meta<UncontrolledToastPlaygroundProps> = {
     isCollapsible: {
       control: 'boolean',
     },
+    enableAutoClose: {
+      control: 'boolean',
+    },
+    autoCloseInterval: {
+      control: 'number',
```

Comment:
can you add default value here?

Reply:
@crishpeen i am not sure what do you mean... there is default value 3000

Reply:
```
      table: {
        defaultValue: { summary: 3000 },
      },

Reply:
default value added